### PR TITLE
Avoid hard-coding the fedcm.json manifest and allow the manifest to b…

### PR DESCRIPTION
**Pull request by samuelgoto**
…e passed directly

***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/278.html" title="Last updated on Jun 16, 2022, 1:07 PM UTC (09cccff)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/278/1a1c446...09cccff.html" title="Last updated on Jun 16, 2022, 1:07 PM UTC (09cccff)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -720,8 +720,8 @@ by exposing a series of HTTP endpoints:
 <!-- ============================================================ -->
 
 The manifest discovery endpoint is an endpoint located at the [=IDP=]'s
```
Comment (npm1) [inline spec/index.bs:722]:
```suggestion
The manifest discovery endpoint is an endpoint
```

File: spec/index.bs [hunk 2]
```diff
@@ -720,8 +720,8 @@ by exposing a series of HTTP endpoints:
...
-`fedcm.json` file and serves as a discovery device to other endpoints provided
-by the [=IDP=].
+and serves as a discovery device to other endpoints provided by the
```
Comment (npm1) [inline spec/index.bs:723]:
```suggestion
which serves as a discovery device to other endpoints provided by the
```

**Review by npm1 — APPROVED** [review]
LGTM with a sentence reword

**Comment by matthewp** [comment]
It's great that this avoids the hardcoding, but I think using `manifest.json` might not be the best choice. Manifest is such an overloaded term, I see manifest.json and immediately [think of this](https://developer.mozilla.org/en-US/docs/Web/Manifest). Can you not use `fedcm.json` in the examples or `fedcm-manifest.json` or something else less generic?

**Pull request merged**